wayrs-client/Cargo.toml

```diff
@@ -26,3 +26,6 @@ features = ["net"]
 # RUSTDOCFLAGS="--cfg docsrs" cargo +nightly doc --all-features --no-deps --open
 all-features = true
 rustdoc-args = ["--cfg", "docsrs"]
+
+[features]
+default = ["tokio"]
```

Comment:
No. Why?

Reply:
I didn't clean the code up properly. I will do this before marking the pr an non-draft.

Reply:
FYI, you can configure rust-analyzer to assume all features are enabled, if this is the reason you did it while developing. In helix it looks like

```toml
[language-server.rust-analyzer.config]
cargo.features = "all"
```